### PR TITLE
teams: new View button

### PR DESCRIPTION
**Pull request by tomholford**
Per the discussion on #22054, the flow for adding a new team member to an org is not intuitive for new Gitea users.

The ideal solution would be to add a new button on the Org > Members index view (see the screenshot mockup in the issue description). However, this would require a refactor of the UX for the flow. The current flow has an implicit context of which team within the org the new member is being added to ('Owners' by default). From the Members index, there is no implicit context; the flow would have to add a picker for which team the new member should be added to.

So, as a stopgap, this change simply adds a button to the Teams index page that performs the same action as clicking on the title of the team (a behavior that is currently too obscure as indicated in the comments on the issue). This should reduce support burden and serve as a decent temporary measure until the Add Member flow is refactored.

### CODE REVIEW

**Comment by Sonic853** [comment]
I think the button here would be best changed to "View". Because this team information page can be viewed by all members of the organization, but only some members with permissions can modify team information and add members on this team information page.

**Comment by tomholford** [comment]
> I think the button here would be best changed to "View". Because this team information page can be viewed by all members of the organization, but only some members with permissions can modify team information and add members on this team information page.

Thanks @Sonic853 for the feedback. I have pushed a commit that changes the copy by adding new translation keys for each language. I am a native English speaker, and I am also familiar with Spanish, Portuguese, and Japanese, so I am comfortable with those translations. For the other languages, I copied the translation from `view_home` and removed the interpolation character (`%s`).

File: options/locale/locale_el-GR.ini [hunk 1]
```diff
@@ -2482,6 +2482,7 @@ members.invite_now=Πρόσκληση Τώρα
 
 teams.join=Συμμετοχή
 teams.leave=Αποχώρηση
+teams.view=Προβολή
```
Comment (lunny) [inline options/locale/locale_el-GR.ini]: Don't translate directly. Use crowdin to translate.

Comment (tomholford) [reply options/locale/locale_el-GR.ini]: @lunny Thanks for the feedback. I will remove the translations from the diff

Comment (tomholford) [reply options/locale/locale_el-GR.ini]: @lunny I just undid the last commit and force-pushed with only the English translation.

File: templates/org/team/teams.tmpl [hunk 2]
```diff
@@ -16,6 +16,7 @@
 					<div class="ui top attached header">
 						<a class="text black" href="{{$.OrgLink}}/teams/{{.LowerName | PathEscape}}"><strong>{{.Name}}</strong></a>
 						<div class="ui right">
+							<a class="btn ui primary small button" href="{{$.OrgLink}}/teams/{{.LowerName | PathEscape}}">{{ctx.Locale.Tr "org.teams.view"}}</a>
```
Comment (wxiaoguang) [inline templates/org/team/teams.tmpl]: There is an exactly same link above: `<a class="text black" href="{{$.OrgLink}}/teams/{{.LowerName | PathEscape}}">`


Why duplicate?

Ideally, either make the existing link more obvious/intuitive (eg: add an icon, or a symbol), or remove the old link, only use the new link.

(just my opinion, not a blocker)

Comment (tomholford) [reply templates/org/team/teams.tmpl]: @wxiaoguang In the linked issue (#22054), multiple users (including myself) didn't realize that the title of the group is also a link. It's only visible on hover. I am not opposed to removing the link. The purpose of this PR is to make it more obvious. Will defer to you, @lunny and @Sonic853 on whether or not we should keep the link in the title.

Comment (wxiaoguang) [inline templates/org/team/teams.tmpl]:
```suggestion
							<a class="ui primary tiny button" href="{{$.OrgLink}}/teams/{{.LowerName | PathEscape}}">{{ctx.Locale.Tr "org.teams.view"}}</a>
```

Then it matches the style of other buttons.

**Review by wxiaoguang — APPROVED** [review]
Overall LGTM. Although I prefer to only keep one link on the UI, I can also accept the duplicate links. So vote my approval ahead.

File: options/locale/locale_en-US.ini [hunk 3]
```diff
@@ -2635,6 +2635,7 @@ members.invite_now = Invite Now
 
 teams.join = Join
 teams.leave = Leave
+teams.view = View
```
Comment (silverwind) [inline options/locale/locale_en-US.ini]: Please move it to top-level translation key `view` here:

https://github.com/go-gitea/gitea/blob/4539a7b0b4fc6539d67ab386eaac4b9aea82bf53/options/locale/locale_en-US.ini#L94

There is nothing about this word that makes it specific to teams. We need to cut down on such useless translations.

**Review by silverwind — CHANGES_REQUESTED** [review]
Translation needs to change

**Comment by tomholford** [comment]
@silverwind @wxiaoguang Thanks for the feedback. I have addressed both requested changes in the latest commit.

**Review by silverwind — APPROVED** [review]

**Pull request merged**